bundles/org.openhab.binding.icalendar/src/main/java/org/openhab/binding/icalendar/internal/handler/ICalendarHandler.java

```diff
@@ -183,6 +183,7 @@ public void onCalendarUpdated() {
                 ThingHandler handler = childThing.getHandler();
                 if (handler instanceof CalendarUpdateListener) {
                     try {
+                        logger.trace("Notifying {} about fresh calendar.", handler.getThing().getUID().getAsString());
```

Comment:
The logger will convert it to a string automatically if it needs to be logged.
```suggestion
                        logger.trace("Notifying {} about fresh calendar.", handler.getThing().getUID());
```